doc/api/GVT-g_api.rst

```diff
@@ -0,0 +1,114 @@
+.. _GVT-g_api:
+
+===============
+ACRN GVT-g APIs
+===============
+
+GVT-g is the GPU virtualization solution of Intel, it is open source and
+up-streamed to the Linux kernel. Its implementation over KVM is named KVMGT,
+over Xen is named XenGT and over ACRN is named ACRN-GT. GVT-g could exports
+multiple virtual GPU (alias vGPU) instances for virtual machine system (a.k.a.
+VM). A VM could be assigned one instance of vGPU. The guest OS graphic driver
+need minor modification to drive the vGPU adapter in VM. Every vGPU instance
+will adopts the full HW GPU’s accelerate capability for 3D render and display.
+In the following document, ACRN-GT refers to the glue layer between ACRN
+hypervisor and GVT-g core device model. It works as the agency of hypervisor
+related services. It is the very and only layer that needs to get rewritten
+when porting GVT-g to other specific hypervisor.  For simplicity, in the rest
+of this document, GVT is used to refer to the core device model component of
+GVT-g, specifically corresponding to gvt.ko when build as module.
+
+Core Driver Infrastructure
+==========================
+
+This section covers core driver infrastructure used by both the display
+and the GEM parts of the driver.
+
+Intel GVT-g Guest Support(vGPU)
+-------------------------------
```

Comment:
Change all uses of ``--------`` to our convention of second level headings using ``=========``